platformer/src/processing/MissionPossible/Level.pde

```diff
@@ -15,12 +15,41 @@ public class Level{
      
      JSONObject json = loadJSONObject(jsonFilePath);
      
-     // Level height, width and background colour
+     // Level height, width, weather and background colour
      levelHeight = json.getInt("height");
      levelWidth = json.getInt("width");
      backgroundColour1 = json.getInt("backgroundColour1");
      backgroundColour2  = json.getInt("backgroundColour2");
      backgroundColour3 = json.getInt("backgroundColour3");
+     String weatherType = json.getString("weather");
+     
+     if(weatherType == "hot"){
+       weather = WeatherVariant.HOT;
+     }
+     else if(weatherType == "windy"){
+       weather = WeatherVariant.WINDY;
+     }
+     else if(weatherType == "icy"){
+       weather = WeatherVariant.ICY;
+     }
+     else if(weatherType == "foggy"){
+       weather = WeatherVariant.FOGGY;
+     }
+     else{
+       weather = WeatherVariant.NEUTRAL;
```

Comment:
You can use a 'switch' statement with strings in Java which would be a bit cleaner here

e.g. [https://www.geeksforgeeks.org/string-in-switch-case-in-java/](url)